src/app/tide/theoretical_peak_set.h

```diff
@@ -1,774 +1,131 @@
-// Benjamin Diament
-
-// A TheoreticalPeakSet is an abstract class representing the collection of
-// theoretical peaks corresponding to a peptide. Its memory is designed to be
-// reusable, so it functions as a "workspace" for figuring where the
-// theoretical peaks ought to be. The expected usage pattern is as follows:
-//
-// TheoreticalPeakSet workspace; // Some construction params may be required.
-// Peptide peptide; // Assume already initialized.
-// workspace.Clear(); // Memory gets reused
-// For each left-substring of peptide, call workspace.AddBIon()
-// For each right-substring of peptide, call workspace.AddYIon()
-//
-// After the B and Y ions have been added the client may retreive the
-// TheoreticalPeakArr by a call to workspace.GetPeaks(). Peaks are then written
-// into the provided arrays (specified in the arguments). To avoid unnecessary
-// copying, for most subclasses, GetPeaks() will take the final steps necessary
-// to generate the peaks and write them directly to the client-specified arrays.
-// There are both "positive" peaks and "negative" peaks that may be written,
-// though some subclasses will never write any negative peaks. When present,
-// these negative peaks represent corrections made to the peaks in the positive
-// set. This comes up in cases where the workspace in question actually
-// represents a difference between two TheoreticalPeakSets. See
-// TheoreticalPeakSetDiff for example.
+// Benjamin Diament and Attila Kertesz-Farkas
```

Comment:
You may want to add a note here that as part of this you removed src/app/tide/theoretical_peak_pair.h from the repo.

Reply:
Good point. Thanks!